valinta-tulos-valintarekisteri-db/src/main/scala/fi/vm/sade/valintatulosservice/valintarekisteri/db/ValinnanTilanKuvausRepository.scala

```diff
@@ -0,0 +1,17 @@
+package fi.vm.sade.valintatulosservice.valintarekisteri.db
+
+import fi.vm.sade.valintatulosservice.valintarekisteri.domain.{HakemusOid, HakukohdeOid, ValinnanTilanKuvausHashCode, ValinnantilanTarkenne, ValintatapajonoOid}
+import slick.dbio.DBIO
+
+trait ValinnanTilanKuvausRepository {
+  def storeValinnanTilanKuvaus(
+                                valinnanTilanKuvausHashCode: ValinnanTilanKuvausHashCode,
+                                hakukohdeOid: HakukohdeOid,
+                                valintatapajonoOid: ValintatapajonoOid,
+                                hakemusOid: HakemusOid,
+                                valinnantilanTarkenne: ValinnantilanTarkenne,
+                                valinnantilanKuvauksenTekstiFI: Option[String],
+                                valinnantilanKuvauksenTekstiSV: Option[String],
+                                valinnantilanKuvauksenTekstiEN: Option[String]
+                      ): DBIO[Unit]
```

Comment:
...ja tässä myös.

Sama pätee noihin muihin vastaaviin.

Parametrien rivittäminen on erittäin jees ja monista paikoista ollaan siivoiltukin jo samaan suuntaan.